website/docs/stitch-combining-schemas.md

```diff
@@ -97,48 +94,112 @@ export interface SubschemaConfig {
+
+See [remote schemas](/docs/remote-schemas/) documentation for more related tools and information.
+
+## Duplicate types
+
+Stitching has two strategies for handling types duplicated across subschemas: a merge strategy (default), and an older binary strategy. You may select between these strategies using the `mergeTypes` option.
 
-An executor is a generic method that performs requests to a remote schema. You may write your own, or use the `linkToExecutor` helper to wrap a link package such as [apollo-link-http](https://www.apollographql.com/docs/link/links/http/). Subschema config accepts an `executor` option for query and mutation operations, and a `subscriber` function for subscription operations. See the [remote schema](/docs/remote-schemas/) docs for more information.
+### Merged types
 
-_**For pre-version 5:** the old method of using [makeRemoteExecutableSchema](/docs/remote-schemas/) to create a local proxy of a remote schema still works. However, it adds an additional layer of delegation that can be avoided by sending settings directly to `stitchSchemas` via SubschemaConfig._
+Duplicate type names are merged by default in GraphQL Tools v7. That means objects, interfaces, and input objects with the same name will have their fields consolidated from across subschemas, and unions will consolidate all member types. The combined gateway schema will then smartly delegate portions of a request to the proper origin schema(s). See [type merging guide](/docs/stitch-type-merging/) for a comprehensive overview.
 
-## Duplicate type definitions
+Type merging will only encounter conflicts on fields and type-level descriptions. By default, the final definition of a field or type description found in the subschemas array is used. You may customize this with `typeMergingOptions` selection logic. For example, the following handlers will select the first non-blank description for each type and field encountered in the subschemas array:
+
+```js
+const gatewaySchema = stitchSchemas({
+  subschemas: [...],
+  mergeTypes: true, // << optional in v7
+  typeMergingOptions: {
+    typeDescriptionsMerger(candidates) {
+      const candidate = candidates.find(({ type }) => !!type.description) || candidates.pop();
+      return candidate.type.description;
+    },
+    fieldConfigMerger(candidates) {
+      const configs = candidates.map(c => c.fieldConfig);
+      return configs.find(({ description }) => !!description) || configs.pop();
+    },
+    inputFieldConfigMerger(candidates) {
+      const configs = candidates.map(c => c.inputFieldConfig);
+      return configs.find(({ description }) => !!description) || configs.pop();
+    }
+  },
+});
+```
+
+### Binary types
```

Comment:
I'm not sure that's the right heading, maybe binary type resolution? But even that is not completely accurate, as users could have been merging types on their own. I'm honestly not sure how all users use the on-type conflict option...

Reply:
Renamed to "Manual resolution"